en/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.rst

```diff
@@ -88,8 +89,8 @@ can start cooking. Use
 :doc:`/console-and-shells/code-generation-with-bake` to quickly
 create your models, controllers, and views.
 
-To use cake bake, call "cake bake all" and this will list the 4
-tables you inserted into mySQL. Select "1. Group", and follow the
+To use cake bake, call ``"cake bake all"`` and this will list the 4
```

Comment:
Done it https://github.com/cakephp/docs/commit/462250b81d6fbe09fde26c13770c128e81fad2f5